assets/src/edit-story/components/canvas/displayLayer.js

```diff
@@ -179,9 +180,11 @@ function DisplayLayer() {
           />
         </DisplayPageArea>
         {enableQuickActionMenu && quickActions.length && (
-          <QuickActionsArea>
-            <ContextMenu isAlwaysVisible isIconMenu items={quickActions} />
-          </QuickActionsArea>
+          <DirectionAware>
+            <QuickActionsArea>
+              <ContextMenu isAlwaysVisible isIconMenu items={quickActions} />
```

Comment:
This is a little bit of a brain teaser - so while the menu itself needs to be `DirectionAware` the tooltips need to reverse to open to the outside edge of the canvas. 🤔 

![Screen Shot 2021-05-18 at 10 31 01 AM](https://user-images.githubusercontent.com/10720454/118697546-43914900-b7c4-11eb-808b-cc2f442ecbc4.png)